packages/fields-wysiwyg-tinymce/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@keystone-alpha/fields-wysiwyg-tinymce",
```

Comment:
Bikeshed: should the name be singular, since there is only one field in this package?

```
"name": "@keystone-alpha/field-wysiwyg-tinymce",
```

Reply:
Good question. I went back and forth myself on the package name a few times.

Settled on `fields` partly just becuase it makes the structure read more coherently in the `packages` directory. Also because some packages may contain multiple field types (e.g I expect `CloudinaryImage` and `CloudinaryImages` would go in the same `fields-cloudinary` package) so it might be nice to keep the naming convention consistent.

Reply:
Even in the `Cloudinary` example, it still feels singular to me; "This field is related to `Cloudinary`.

BUT! I don't really care all that much. We just need to make sure we're consistent going forward 👍